software/prepare_src/include/estream-convert-api.h

```diff
@@ -0,0 +1,118 @@
+/*
```

Comment:
Could you please specify which AEAD or hash scheme depend on each of the added header files? How many of the schemes now compile and run without modification using `cryptotvgen`? Which ones still don't?